db/migrate/20200219152902_add_meta_fields_to_team_member.rb

```diff
@@ -1,6 +1,6 @@
 class AddMetaFieldsToTeamMember < ActiveRecord::Migration[6.0]
   def change
-    add_column :team_members, :role, :string
+    add_column :team_members, :discipline, :string
```

Comment:
You can't edit a migration that's already run on your live environment. Migration only run once, so your live version will have already run this migration. Instead, you need to create a new migration to do a rename of the column.

However, this PR is about styling the frontend. Renaming the database field probably belongs in a different commit.

Reply:
OK. What I did was rolling the DB back to before I ran this file, edit the file and then run `rails db:migrate` again. That's what I understood from the previous suggestions on this PR. 

Shall I create a new PR with this commit then?